languages/swift/exercises/concept/strings-and-characters/.docs/introduction.md

```diff
@@ -0,0 +1,111 @@
+// Error: Cannot convert value of type 'String' to specified type 'Character'
+```
+
+Like other constructs in Swift, characters and strings are immutable if defined using `let` and mutable if defined using `var`. So in the above examples, `hello` and `sausage` are immutable strings while `fam` and `empty`, AND `aChar` are mutable.
+
+### Building strings from smaller parts
+
+Strings can be concatenated using the `+` operator, and mutable strings can have other strings appended onto them using the `+=` mutating operator.
+
+```swift
+"honey" + "comb"
+// => "honeycomb"
+
+fam += ", this is our cat: 🐱"
+// => "This is my family: 👨‍👩‍👦‍👦, this is our dog: 🐶, this is our cat: 🐱"
+```
+
+Note that only strings can be concatenated with strings. In order to use concatenation with a character, one must convert it to a String first
+
+```swift
+let question: Character = "?"
+let areYouHappy = "Happy" + question
+// Error: Cannot convert value of type 'Character' to expected argument type 'String'
+
+let areYouHappy = "Happy" + String(question)
+// => "Happy?"
+```
+
+### String Interpolation
+
+The most common way to build up strings in Swift is _string interpolation_. To insert a value into a string using string interpolation, you place an expression or value in parentheses which are preceded by a backslash character.
+
+```swift
+let radius = 5.0
+let interp = "The area of a circle with radius \(radius) is \(Double.pi * radius * radius)"
+// => "The area of a circle with radius 5.0 is 78.53981633974483"
+```
+
+### String and Character equality
+
+Strings and Characters can be compared for equality (or lack thereof) using the `==` operator (or `!=` for not-equal).
+
+```swift
+let hi = "Hello"
+let period: Character = "."
+"\(hi)\(period)" == "Hello."
+// => true
+
+period != ";"
+// => true
+```
+
+### String and Character properties
+
+Strings and characters have a few different _properties_ which can be queried to get information about the string or character. These are queried by placing a ._propertyName_ after the string or character in question For example, one can check to see if a string is empty by querying it's `isEmpty` property, and the count of characters in a string can be retrieved using its `count` property.
+
+Character properties are mostly based on the type of character it is. For example, one can check to see if a character `isNumber`, `isLetter`, `isUppercase`, etc.
+
+```swift
+empty.isEmpty
+// => true
+fam.count
+// => 60
+aChar.isLowercase
+// => false
+let half: Character = "½"
+half.isNumber
+// => true
+```
+
+### Iterating over the characters of a string
+
+The individual characters in a string can be stepped through one at a time using a for-in loop. This type of loop takes each character in the string, in order, and binds the character to a specified name for further processing inside the loop body. For example, to print out all of the capital letters in a string one can write:
```

Comment:
This exercise introduces a `for-in` loop. I would suggest adding this to the `concepts` unlocked by the exercise, as I assume this is something you want to depend on later?

Reply:
I will be removing this part as there are no good motivating examples without the conditionals, plus it is covered in more depth in the `loops` exercise a couple of steps up the road.

Reply:
Excellent